tests/aws/services/cloudformation/resources/test_dynamodb.py

```diff
@@ -163,3 +163,45 @@ def test_ttl_cdk(aws_client, snapshot, infrastructure_setup):
         table_name = outputs["TableName"]
         table = aws_client.dynamodb.describe_time_to_live(TableName=table_name)
         snapshot.match("table", table)
+
+
+@markers.aws.validated
+@markers.snapshot.skip_snapshot_verify(
+    [
+        "$..Table.ProvisionedThroughput.LastDecreaseDateTime",
+        "$..Table.ProvisionedThroughput.LastIncreaseDateTime",
+        "$..Table.Replicas",
+        "$..Table.SSEDescription.KMSMasterKeyArn",
```

Comment:
This should be a transformer, rather than a skip here. If I execute this in my account, this test will not pass, since anything in  `skip_snapshot_verify` is disabled when testing against AWS.

Reply:
Fixed